src/diagnostics.rs

```diff
@@ -233,138 +236,136 @@ impl Message {
-                } else {
-                    formatter.get_actual_column(span.start.col)
-                }),
-                markers = "^".repeat(
-                    formatter.get_actual_column(span.end.col + 1)
-                        - formatter.get_actual_column(span.start.col)
-                )
-            );
-
-            output.set_bold(true);
-            output.set_color(self.level.color());
-            writeln!(output.writer(), "{}", indicator);
-        } else {
-            // move position to first element in line as we always want to
-            // render the first N characters if a line is way to long for
-            // an error message.
-            // TODO: would be nicer to always show the column of a multiline
-            // error. But I am not sure how the line columns should be aligned
-            // if lines are truncated differently.
-            let mut line_first_char = span.start.to_line_start();
-
-            for _line_num in span.start.row..=span.end.row {
-                // add line number
-                output.set_color(HIGHLIGHT);
-                output.set_bold(true);
-                write!(
-                    output.writer(),
-                    " {padded_linenumber} |",
-                    padded_linenumber =
-                        pad_left(&(line_first_char.row + 1).to_string(), line_number_width)
-                );
-
-                // Add marker at the beginning of the line, if it spans
-                // multiple lines
-                output.set_color(self.level.color());
-                write!(output.writer(), "> ");
-
-                // add source code line
-                output.set_color(None);
-                output.set_bold(false);
-
-                let (truncation_before, src_line, truncation_after) =
-                    line_first_char.get_line(1, MAX_CONTEXT_LENGTH_MULTILINE);
-                debug_assert!(truncation_before == LineTruncation::NotTruncated);
-
-                let formatter = LineFormatter::new(&src_line);
-                formatter.render(output.writer());
-
-                if truncation_after == LineTruncation::Truncated {
-                    output.set_color(HIGHLIGHT);
-                    output.set_bold(true);
-                    write!(output.writer(), "{}", truncation_str);
-                }
-
-                writeln!(output.writer());
-
-                line_first_char = match line_first_char.next_line() {
-                    Ok(pos) => pos,
-                    Err(_) => {
-                        /* EOF */
-                        break;
-                    }
-                }
-            }
-
-            // add padding line below
-            output.set_color(HIGHLIGHT);
-            output.set_bold(true);
-            writeln!(output.writer(), "{}", empty_line_marker);
-        }
+        //let line_number_width = (span.end.row + 1).to_string().len();
```

Comment:
comment commited

Reply:
see comment above. This is temp code to make it compile until the code is rewritten.